boards/ikea-tradfri/Makefile.features

```diff
@@ -10,5 +10,8 @@ FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart periph_uart_modecfg
 
+# Various other features (if any)
+FEATURES_PROVIDED += emulator_renode
```

Comment:
I see this at multiple places, but it doesn't seem to be used, right?

Reply:
I checked out your code. I don't know. I guess not.

Reply:
Sorry for 'blaming' you, I actually found out I wrote that myself in #8153 :-P